releasenotes/notes/no-useless-DNS-queries-1ed8ff14e0684da1.yaml

```diff
@@ -6,6 +6,12 @@
 #
 # Each section note must be formatted as reStructuredText.
 ---
+upgrade:
+  - |
+    In order to avoid unnecessary DNS queries, the agent now uses FQDN when connecting to Datadog intakes.
+    Specifically, it adds a trailing dot at the end of the Datadog intake hostnames.
+While most users may not notice this change, it can affect setups where connections between the agent and Datadog intakes are intercepted for deep pocket inspection or TLS man-in-the-middle by proxies or firewalls.
```

Comment:
```suggestion
    While most users may not notice this change, it can affect setups where connections between the agent and Datadog intakes are intercepted for deep packet inspection or TLS man-in-the-middle by proxies or firewalls.
```